lib/test_track.rb

```diff
@@ -52,6 +54,27 @@ def misconfiguration_notifier_class_name=(notifier_class_name)
       @misconfiguration_notifier_class_name = notifier_class_name
     end
 
+    def load_build_timestamp # rubocop:disable Metrics/MethodLength
+      if Rails.env.test? || Rails.env.development?
+        @build_timestamp = Time.zone.now.iso8601
+      elsif File.exist?(BUILD_TIMESTAMP_FILE_PATH)
+        timestamp = _build_timestamp
+
+        unless BUILD_TIMESTAMP_REGEX.match?(timestamp)
+          raise "./testtrack/build_timestamp is not a valid ISO 8601 timestamp, got '#{timestamp}'"
+        end
+
+        @build_timestamp = timestamp
+      else
+        raise 'TestTrack failed to load the required build timestamp. ' \
+          'Ensure `test_track:generate_build_timestamp` task is run and the build timestamp file is present.'
+      end
+    end
+
+    def build_timestamp
+      @build_timestamp.presence || raise('build_timestamp is not defined because `load_build_timestamp` initializer was not run.')
```

Comment:
I wonder if this should note that it normally runs as part of `assets:precompile`

Reply:
(same as the error above)